.github/ISSUE_TEMPLATE/new-gallery-example-request.md

```diff
@@ -0,0 +1,29 @@
+---
+name: New gallery example request
+about: This template is for requests for new examples
+title: 'New example to show how to <add words here>'
+labels: new example request
```

Comment:
I can't seem to find a reference on what the synatx for this is, I assume this is going to add one label and not three?!

Reply:
I cheated and used the template tool from GitHub so I assume this is right.

Reply:
cool